packages/protoplugin/src/ecmascript/parameter.ts

```diff
@@ -120,21 +118,6 @@ export function parseParameter(
             throw new PluginOptionError(raw);
         }
         break;
-      case "keep_empty_files": {
```

Comment:
Users who set this option in their buf.gen.yaml will get an error with this change, and they'll wonder why. Can we provide a helpful error message in this case? I guess that's a bit tricky, because we'll no longer support this option, but plugin users can't use it either if we error on it.

Reply:
I think this should be handled as part of the upgrade guide.